Makefile

```diff
@@ -0,0 +1,101 @@
+PYTHON	= $(shell which python)
+
+TOPDIR  = $(shell pwd)
+PYDIR	= koku
```

Comment:
`masu`?

Reply:
😴 @ the ☸️ again, clearly. Thanks for catching these.